Firmware/ultralcd.cpp

```diff
@@ -7427,3 +7437,52 @@ void lcd_heat_bed_on_load_toggle()
         value = !value;
     eeprom_update_byte((uint8_t*)EEPROM_HEAT_BED_ON_LOAD_FILAMENT, value);
 }
+
+void reprint_from_eeprom() {
+	char filename[13];
+	char altfilename[13];
+	uint8_t depth = 0;
+	char dir_name[9];
+
+	enableReprint=false;
+
+	//cmdqueue_reset();
+
+	depth = eeprom_read_byte((uint8_t*)EEPROM_DIR_DEPTH);
+
+	MYSERIAL.println(int(depth));
+	for (int i = 0; i < depth; i++) {
+		for (int j = 0; j < 8; j++) {
+			dir_name[j] = eeprom_read_byte((uint8_t*)EEPROM_DIRS + j + 8 * i);
+		}
+		dir_name[8] = '\0';
+		MYSERIAL.println(dir_name);
```

Comment:
Minor nitpick. But I feel all the calls to `MYSERIAL` in this function should be removed. The increase flash memory usage unnecessarily.

In my opinion they are local debugging code.

```c
MYSERIAL.println(int(depth));
MYSERIAL.println(dir_name);
MYSERIAL.print(altfilename);
```

Reply:
It was for debug purpose, already deleted

Reply:
Sorry for picking at these small things but there are two `MYSERIAL` lines left.

Reply:
> Sorry for picking at these small things but there are two `MYSERIAL` lines left.

I'm sorry, I have removed already.

Reply:
Thanks :) Resolving this thread. 

I have yet to test the PR on hardware, will approve later this week if all works as expected.

Reply:
Maybe would be nice to check the translations of "REPRINT" because I used google translator